drivers/sensor/ltrf216a/CMakeLists.txt

```
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+
+zephyr_library_sources_ifdef(CONFIG_LTR_F216A ltrf216a.c)
```

It's redundant to check `CONFIG_LTR_F216A` since it was already checked in `drivers/sensor/CMakeLists.txt`
```suggestion
zephyr_library_sources(ltrf216a.c)
```

---

applied in the latest force push.